source/api/presentation/3.0/index.md

```diff
@@ -728,24 +761,25 @@ The value _MUST_ be a JSON object, which _MUST_ have the `id` and `type` propert
 { "includes": { "id": "https://example.org/iiif/1/annos/1", "type": "AnnotationCollection" } }
 ```
 
-### 3.5. Structural Properties
+### 3.4. Structural Properties
 
 These properties define the structure of the object being represented in IIIF by allowing the inclusion of child resources within parents, such as a Canvas within a Manifest, or a Manifest within a Collection. The majority of cases use `items`, however there are two special cases for different sorts of structures.
 
 ##### items
 
-Much of the functionality of the IIIF Presentation API is simply recording the order in which child resources occur within a parent resource, such as Collections or Manifests within a parent Collection, or Canvases within a Manifest. All of these situations are covered with a single property, `items`.  
-The value _MUST_ be an array of JSON objects. The items will be resources of different types, as described below.
+Much of the functionality of the IIIF Presentation API is simply recording the order in which child resources occur within a parent resource, such as Collections or Manifests within a parent Collection, or Canvases within a Manifest. All of these situations are covered with a single property, `items`. 
+
+The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and `type` properties. The items will be resources of different types, as described below.
 
 * A Collection _MUST_ have the `items` property. Each item _MUST_ be either a Collection or a Manifest.<br/>
   Clients _MUST_ process `items` on a Collection.
-* A Manifest _MUST_ the `items` property. Each item _MUST_ be a Canvas.<br/>
+* A Manifest _MUST_ have the `items` property with at least one item. Each item _MUST_ be a Canvas.<br/>
   Clients _MUST_ process `items` on a Manifest.
-* A Canvas _SHOULD_ have the `items` property. Each item _MUST_ be an Annotation Page<br/>
+* A Canvas _SHOULD_ have the `items` property with at least one item. Each item _MUST_ be an Annotation Page<br/>
```

Comment:
Another thought here - probably a non-starter as it may cause more problems than it solves.

We're likely (as with sequence) to have a 99.99..% case where the content is always in `canvas.items[0].items`, especially as only "painting" annos are allowed ("transcribing" go in `annotations`). So,...

Could a Canvas _be_ an Annotation Page, so its `items` property contains annotations directly (like P2 `canvas.images`)?

Pros - another layer removed from the model; more like P2 (or items as canvas.content rather than canvas.pagesofcontent)
Cons - can't partition `canvas.items` into separately labelled pages, with one or more external to the manifest; possible other problems if a Canvas is also an AnnotationPage

What scenarios would result in a very large number of painting annos, that couldn't be embedded?

This is another issue if there are any legs in it.

Reply:
If it were a page, then it couldn't have other pages... so everything would be one big array of annotations. :-1:

Reply:
But fixed missing `.`